lib/bolt/util.rb

```diff
@@ -77,6 +77,14 @@ def read_optional_yaml_hash(path, file_name)
         File.exist?(path) ? read_yaml_hash(path, file_name) : {}
       end
 
+      def first_runs_free
+        Pathname.new(File.expand_path(File.join('~', '.puppetlabs', 'etc', 'bolt', '.first_runs_free')))
```

Comment:
This will blow up if there's no homedir. We should use [`Bolt::Config.user_path`](https://github.com/puppetlabs/bolt/blob/main/lib/bolt/config.rb#L73) and handle the case where that returns `nil`.